officers.html

```diff
@@ -137,7 +137,31 @@ <h2>Nate Hood</h2>
                                     </div>
                                 </div>
                             </div>
+
+                            <div class="column">
+                                <div class="card">
+                                    <img src="images/officers/thomas.jpg" alt="John" style="width:100%" />
+                                    <div class="card-text">
+                                        <h2>Thomas Stansel</h2>
+                                        <!-- Nothing Is on my Website right now
+                                        <p class="title">VP Events</p>

```

Comment:
did you mean to comment out your title?

Reply:
going to merge, but you may want to uncomment your title so that shows up in the card